server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -184,30 +184,6 @@ public void getMessageListShouldNotLeastMessageIfTheUserHasOnlyReadRight() throw
             .body(ARGUMENTS + ".messageIds", hasSize(0));
     }
 
-    public void getMessageListShouldNotLeastMessageIfTheUserHasOnlyLookupRight() throws Exception {
```

Comment:
why removing this test?

Reply:
There is these two tests on the final PR:

 - getMessageListShouldListMessagesWhenReadRight
 - getMessageListShouldNotListMessageIfTheUserHasOnlyLookupRight

Luc PR was originally was originaly done on master, which was failing, hence the bad behaviour, and strange right combination (r+l, r, l), which is no longer required (r, pas r)

Reply:
ok but what happens if the user has only lookup rights? this test is still interesting, no?

Reply:
It happens this!

```
+
+    @Test
+    public void getMessageListShouldNotListMessageIfTheUserHasOnlyLookupRight() throws Exception {
+        mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, bob, "delegated");
+        MailboxPath delegatedMailboxPath = MailboxPath.forUser(bob, "delegated");
+        mailboxProbe.appendMessage(bob, delegatedMailboxPath,
+            new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()), new Date(), false, new Flags());
+
+        await();
+
+        aclProbe.replaceRights(delegatedMailboxPath,
+            alice,
+            new Rfc4314Rights(Right.Lookup));
+
+        given()
+            .header("Authorization", aliceAccessToken.serialize())
+            .body("[[\"getMessageList\", {}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("messageList"))
+            .body(ARGUMENTS + ".messageIds", empty());
+    }
```

Directly picked from the diff ;-)

Reply:
and so? why no keeping it?

Reply:
We have it!

![capture d ecran de 2017-10-16 17-42-56](https://user-images.githubusercontent.com/6928740/31608164-8480bc5e-b299-11e7-94af-41b89c8f35e6.png)

The test is there, your comment is on an outdated commit. Please read more carefully.